mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature

```diff
@@ -14,3 +15,37 @@ Feature: linking-rules tests
     And assert karate.sizeOf(response[0].authoritySubfields) > 0
     And assert response[0].bibField != null
     And assert response[0].authorityField != null
+    And assert response[0].autoLinkingEnabled != null
+
+  @Positive
+  Scenario: Get instance to authority rules by id - Should return appropriate rule
+    Given path '/linking-rules/instance-authority/' + defauldLinkId
+    When method GET
+    Then status 200
+    And assert response != null
+    And assert response.bibField == 100
+    And assert response.authorityField == 100
+    And assert response.autoLinkingEnabled != null
+    And assert karate.sizeOf(response.authoritySubfields) > 0
+
+  @Positive
+  Scenario: Patch linking rule by id - Should update appropriate rule
+    Given path '/linking-rules/instance-authority/' + defauldLinkId
```

Comment:
Please add more scenarios:
postive:
trying to update all fields of a rule -> only autoLinkingEnabled is updated

negative: 
rule not found -> 404